test/DebugInfo/ASTSection_ObjC.swift

```diff
@@ -12,6 +12,9 @@
 
 // REQUIRES: executable_test
 // REQUIRES: objc_interop
+// UNSUPPORTED: OS=linux-gnu
+// UNSUPPORTED: OS=linux-gnueabihf
+// UNSUPPORTED: OS=freebsd
```

Comment:
Are these OS's not already covered by the `REQUIRES: objc_interop` requirement above? I believe #1032 disabled this test for all Linux variants.